### PR TITLE
Feed fetch default true

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
regarding:
https://github.com/demisto/etc/issues/27176

## Description
edited the required params so feed integrations will require fetch true in default value.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -854,6 +854,7 @@ class PB_Status:
 
 FEED_REQUIRED_PARAMS = [
     {
+        'defaultvalue': 'true',
```
Comment (yaakovi) [inline demisto_sdk/commands/common/constants.py:857]: shouldn't this be a boolean?

**Comment by orhovy** [comment]
@yaakovi 
ran validate already on all repo and updated the failures.
PR: https://github.com/demisto/content/pull/8367

File: demisto_sdk/commands/common/constants.py (hunk 1 above)
Comment (orhovy) [reply demisto_sdk/commands/common/constants.py:857]: In some integration it
defaultvalue: true
and in some its
defaultvalue: 'true'
Both work.
I checked how the majority looks and its that way. defaultvalue: 'true'

Comment (yaakovi) [reply demisto_sdk/commands/common/constants.py:857]: seems odd that the default value is string when the type of the parameter is boolean but ok...
if that's the majority then let's keep it that way.

**Comment by orhovy** [comment]
image
ran on master and validated all files

**Pull request merged**